### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by andela-hthuo**
#### What does this PR do?
Updates the README to be more comprehensive.

#### Description of Task to be completed?
Add to the following to the README:
 * A description of the project
 * How to run the project
 * How to run the tests
 * Technologies used in the project
 * License

#### How should this be manually tested?
Read the updated README [here](https://github.com/andela/trime/blob/chore/update-readme/README.md).

#### What are the relevant pivotal tracker stories?
[Add steps to setup the project on a new machine](https://www.pivotaltracker.com/projects/1454736/stories/117876021)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,108 @@
-# trime
+# Trime
 
 [![Code Climate](https://codeclimate.com/github/andela/trime/badges/gpa.svg)](https://codeclimate.com/github/andela/trime) [![Test Coverage](https://codeclimate.com/github/andela/trime/badges/coverage.svg)](https://codeclimate.com/github/andela/trime/coverage) [![Coverage Status](https://coveralls.io/repos/github/andela/trime/badge.svg?branch=feature%2Fbendtest-coverage)](https://coveralls.io/github/andela/trime?branch=feature%2Fbendtest-coverage)
```
Comment (andela-kndungu) [inline README.md]: Remove the second test coverage badge, it's inactive.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,108 @@
...
 [![codecov.io](https://codecov.io/github/andela/trime/coverage.svg?branch=master)](https://codecov.io/github/andela/trime?branch=master)
 
-trime project files
-Time Tracking application.
-Track #start, #create and #finish organization projects.
+Trime aims to add value to companies by helping them to account for time spent working on projects.
+
+It's a flexible and easy to use solution that helps you and your colleagues keep track of time spent on projects.
+
```
Comment (andela-kndungu) [inline README.md]: Make it more... descriptive

File: README.md [hunk 3]
```diff
@@ -1,8 +1,108 @@
...
+```
+
+**`cd` into the project directory:**
+```
+cd trime
+```
+
+**Set up the database:**
+
+Create a Postgres database called `trime` to use with the app.
```
Comment (andela-kndungu) [inline README.md]: How?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,108 @@
...
+
+**Set the environment variables:**
+
+Create a .env file with the following contents
+```
+DATABASE_NAME = 'trime'
+DATABASE_DIALECT = 'postgres'
+DATABASE_HOST = 'localhost'
+DATABASE_USERNAME = 'YOUR_USERNAME'
```
Comment (andela-kndungu) [inline README.md]: Currently it looks like just a string that should be copy pasted. Add a comment making it clear that one should replace 'YOUR_USERNAME' with their own username

File: README.md [hunk 5]
```diff
@@ -1,8 +1,108 @@
...
+GITHUB_CLIENT_ID = 'YOUR_GITHUB_CLIENT_ID'
+GITHUB_CLIENT_SECRET = 'YOUR_GITHUB_CLIENT_SECRET'
+GOOGLE_CLIENT_ID = 'YOUR_GOOGLE_CLIENT_ID'
+GOOGLE_CLIENT_SECRET = 'YOUR_GOOGLE_CLIENT_SECRET'
+SECRET_KEY = 'YOUR_SECRET_KEY'
+```
+
+**Install dependencies:**
+```
+npm install
```
Comment (andela-kndungu) [inline README.md]: You need to make it clear that this command must be run inside the project folder

File: README.md [hunk 6]
```diff
@@ -1,8 +1,108 @@
...
+```
+
+
+**Run:**
+
+Start Postgres before running using gulp
```
Comment (andela-kndungu) [inline README.md]: How?

File: README.md [hunk 7]
```diff
@@ -1,8 +1,108 @@
...
+```
+
+Open `http://localhost:3000` on your browser to view the app.
+
+
+## Running the tests
+
+Run:
+```
+npm test
```
Comment (andela-kndungu) [inline README.md]: Make it clear that one must be inside the project folder.

File: README.md [hunk 8]
```diff
@@ -1,8 +1,108 @@
...
+```
+npm run test:fend
+```
+
+### Back end tests
+
+Test the backend only
+
+```
+npm run test:fend
```
Comment (andela-kndungu) [inline README.md]: Typo

File: README.md [hunk 9]
```diff
@@ -1,8 +1,108 @@
...
+cd trime
+```
+
+**Set up the database:**
+
+Create a Postgres database called `trime` to use with the app.
+
+**Set the environment variables:**
+
+Create a .env file with the following contents
```
Comment (andela-kndungu) [inline README.md]: Where and how?

Suggestion
screen shot 2016-05-11 at 4 52 11 pm

File: README.md [hunk 10]
```diff
@@ -1,8 +1,108 @@
...
+
+## Getting Started
+
+### Prerequisities
+
+* [Node.js](https://nodejs.org/en/download/)
+* [Postgres](http://www.postgresql.org/download/)
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* Gulp - `npm install -g gulp`
+
```
Comment (andela-kndungu) [inline README.md]: Suggestion
screen shot 2016-05-11 at 4 49 50 pm

File: README.md (hunk 2 above)
Comment (andela-jobala) [inline README.md]: Your suggestion is vague. It would help to point out issues you feel we left out.

**Pull request merged**